content/docs/guides/troubleshooting.md

```diff
@@ -172,3 +172,12 @@ ks version
 ```
 
 If your ksonnet version is lower than what is specified in the [requirements](/docs/guides/requirements), please upgrade it and follow the [guide](/docs/guides/components/ksonnet) to recreate the app.
+
+## ksonnet on Windows
+There are some known issues with ksonnet and Windows. You might consider 
+alternative solutions.
+
+ * construct base object: Failed to filter components; the following 
+ components don't exist:  ['kubeflow-core'] [kubeflow #481](https://github.com/kubeflow/kubeflow/issues/481)
```

Comment:
This link is [rendering oddly now](https://deploy-preview-216--competent-brattain-de2d6d.netlify.com/docs/guides/troubleshooting/) - it looks like this on the page:
[‘kubeflow-core’] #481

Suggested markup and reduced text:

```
 * construct base object: Failed to filter components ([kubeflow #481](https://github.com/kubeflow/kubeflow/issues/481))
```